week-1/Homework/extra/2-piping.js

```diff
@@ -16,43 +16,50 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  return (num1 * 10 + num2 * 10) / 10;
 }
 
-function multiply() {
+function multiply(num1, num2) {
+  return num1 * num2;
+}
 
+function format(num) {
+  return "£" + num;
 }
 
-function format() {
+const startingValue = 2;
 
+function number() {
+  const x = startingValue + 10;
+  const y = x * 2;
+  return "£" + y;
 }
 
-const startingValue = 2
-
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode = 
+let badCode = number();
 
 /* BETTER PRACTICE */
-
-let goodCode = 
-
+function better() {
+  return "£" + (startingValue + 10) * 2;
+}
+let goodCode = better();
```

Comment:
This is also good, but you can still use functions you have created before (call function inside a function)